org.scala-ide.sdt.core/src/org/scalaide/extensions/autoedits/SurroundBlock.scala

```diff
@@ -92,10 +102,8 @@ trait SurroundBlock extends AutoEdit {
 
             if (prevLine.start == firstLine.start)
               None
-            else if (t.tokenType == Tokens.VARID)
```

Comment:
I can't remember for what that case was needed. It is very well possible that it is no longer needed.